src/test/resources/org/broadinstitute/hellbender/tools/walkers/VQSR/expected.AS.alternate.tranches

```diff
@@ -1,15 +1,15 @@
 # Variant quality score tranches file
 # Version number 5
 targetTruthSensitivity,numKnown,numNovel,knownTiTv,novelTiTv,minVQSLod,filterName,model,accessibleTruthSites,callsAtTruthSites,truthSensitivity
-90.00,0,2507,0.0000,3.2205,-1.4017,VQSRTrancheSNP0.00to90.00,SNP,2786,2507,0.8999
-97.00,0,2702,0.0000,3.2351,-2.1306,VQSRTrancheSNP90.00to97.00,SNP,2786,2702,0.9698
-98.00,0,2730,0.0000,3.2457,-2.4533,VQSRTrancheSNP97.00to98.00,SNP,2786,2730,0.9799
-99.00,0,2758,0.0000,3.2496,-3.2164,VQSRTrancheSNP98.00to99.00,SNP,2786,2758,0.9899
-99.30,0,2766,0.0000,3.2488,-4.0653,VQSRTrancheSNP99.00to99.30,SNP,2786,2766,0.9928
-99.40,0,2769,0.0000,3.2469,-5.1648,VQSRTrancheSNP99.30to99.40,SNP,2786,2769,0.9939
-99.50,0,2772,0.0000,3.2450,-5.9262,VQSRTrancheSNP99.40to99.50,SNP,2786,2772,0.9950
-99.60,0,2774,0.0000,3.2416,-6.5841,VQSRTrancheSNP99.50to99.60,SNP,2786,2774,0.9957
-99.80,0,2780,0.0000,3.2314,-58913.3494,VQSRTrancheSNP99.60to99.80,SNP,2786,2780,0.9978
-99.90,0,2783,0.0000,3.2295,-59210.6252,VQSRTrancheSNP99.80to99.90,SNP,2786,2783,0.9989
-100.00,0,2786,0.0000,3.2340,-Infinity,VQSRTrancheSNP99.90to100.00,SNP,2786,2786,1.0000
```

Comment:
Probably doesn't happen in real runs, but these ties and the corresponding ordering funkiness aren't great, IMO...

Reply:
Yes, the "accessible truth sites" are the same (which may always be true? I don't remember), but I don't see what's weird about the ordering.

Reply:
Just the fact that the 99.95 target truth sensitivity appears last, after 100. I would guess that’s because the VQSLODs are tied at -Infinity and there’s nothing else in the comparator to indicate the natural ordering.

Reply:
Oh in the old one?  I guess I never noticed that.  I don't think that's worth fixing today.